pkgs/development/python-modules/scipy/default.nix

```diff
@@ -107,7 +108,18 @@ in buildPythonPackage {
 
   __darwinAllowLocalNetworking = true;
 
-  nativeCheckInputs = [ nose pytest pytest-xdist ];
+  nativeCheckInputs = [ nose pytest pytest-xdist pytestCheckHook ];
```

Comment:
Why do you need `pytestCheckHook` here? Or alternatively, perhaps you could put `pytestCheckHook` instead of `pytest`, and use `dontUsePytestCheck = true;`.

Reply:
I include the hook to use `_pytestComputeDisabledTestsString` in the check phase. Since I’m already copying most of the hook’s logic anyway, should I also just copy that as well into the check phase?

Reply:
> I include the hook to use `_pytestComputeDisabledTestsString` in the check phase. Since I’m already copying most of the hook’s logic anyway, should I also just copy that as well into the check phase?

Mmmm that's not what I meant, I don't mind the hook being there, I only mind that both `pytest` and `pytestCheckHook` are there. Now after reading more carefully what's going on, the solution proposed is pretty much the best there could be.

Reply:
```
  nativeCheckInputs = [
    nose
    pytestCheckHook
    pytest-xdist
  ];
```

Reply:
Oh, right. I updated the commit per your suggestion to drop pytest from the `nativeCheckInputs` (since it’s propagated by the hook).